doc/dev/debugging.rst

```diff
@@ -120,33 +120,8 @@ Pass Pipelines
 
 The compilation steps which take MLIR as an input and lower it to binary are broken into MLIR pass
 pipelines.  The ``pipelines`` argument of the ``qjit`` function may be used to alter the steps used
-for compilation. The default set of pipelines is defined via the ``catalyst.compiler.DEFAULT_PIPELINES``
-list. Its structure is shown below.
-
-.. code-block:: python
-
-    DEFAULT_PIPELINES = [
-        (
-            "HLOLoweringPass",
-            [
-                "canonicalize",
-                "func.func(chlo-legalize-to-hlo)",
-                "stablehlo-legalize-to-hlo",
-                "func.func(mhlo-legalize-control-flow)",
-                ...
-            ],
-        ),
-        (
-            "QuantumCompilationPass",
-            [
-                "lower-gradients",
-                "adjoint-lowering",
-                "convert-arraylist-to-memref",
-            ],
-        ),
-        ...
-        ]
-
+for compilation. The default set of pipelines is defined via the ``catalyst.compiler.get_stages``
+list.
```

Comment:
Is this the right function? Should it be?
```suggestion
for compilation. The default set of pipelines is defined via the :func:`~.get_compilation_stage`
list.
```

Reply:
It is correct. I just defined that in this PR.

Reply:
sorry, I shouldn't speed review 🤦